skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevbehandler/-components/BrevbehandlerMeny.tsx

```diff
@@ -163,13 +142,13 @@ const ArkivertBrev = (props: { brev: BrevInfo }) => {
       <div>
         <Detail textColor="subtle">Mottaker</Detail>
         {props.brev.mottaker ? (
```

Comment:
Her trenger du kanskje ikke å ha et tertiært uttrykk?  Eneste forskjellen mellom de to er at `props.brev.mottaker` er `undefined` i stedenfor `null`. Samme litt lenger ned i filen her.